### DISCUSSION TITLE
Nested Queries with clients

### DISCUSSION THREAD

**Discussion by Prashant4900** [supabase/supabase#2686] [Questions]
select * from shop_product where updated_at in
(select distinct updated_at from shop_product order by updated_at limit 1)

Sir how we written this type of nested queries with clients?

**Comment by silentworks** [comment]
You generally don't write complex queries with the client, this type of query is better created as a View or a database function (only use a function if you are passing a parameter to it). You can read more about postgres database views on the following website https://www.postgresqltutorial.com/postgresql-views/
